src/vs/workbench/api/common/apiCommands.ts

```diff
@@ -32,7 +32,7 @@ function adjustHandler(handler: (executor: ICommandsExecutor, ...args: any[]) =>
 
 interface INewWindowAPICommandOptions {
 	reuseWindow?: boolean;
-	remoteAuthority?: string;
+	remoteAuthority?: string | null; /* allow null to force a local window */
```

Comment:
Again, JSDoc please and maybe copy the JSDoc from the internal API if it applies here.